locations.json

```diff
@@ -30,5 +30,15 @@
     "admin0": "United States",
     "admin1": "Massachusetts",
     "admin1_abbr": "MA"
+  },
```

Comment:
This is fixed since locations.json went away in the meantime. Github may or may not show it as changed, but it's not changed in the final version of this branch.